builtInFunctions/esdtRoles.go

```diff
@@ -134,14 +154,21 @@ func (e *esdtRoles) CheckAllowedToExecute(account vmcommon.UserAccountHandler, t
 	if isNew {
 		return ErrActionNotAllowed
 	}
+	if !searchForRole(roles, action) {
+		return ErrActionNotAllowed
+	}
+
+	return nil
+}
 
+func searchForRole(roles *esdt.ESDTRoles, action []byte) bool {
```

Comment:
could rename this function to `isRoleSet()` or `isActionAllowed()` as the function returns a boolean

Reply:
deleted as we had another function which did the same